File: Basic_Algorithm/801/sum_of_one.cpp
#include <iostream>
using namespace std;

int n, t, res;

int main() {
    scanf("%d", &n);
    while (n--) {
        scanf("%d", &t);
        res = 0;
        while (t) {
            res += t & 1;
            t >>= 1;
        }
        printf("%d ", res);
    }
    return 0;
}

File: Basic_Algorithm/801/sum_of_one.py
if __name__ == "__main__":
    n = int(input())
    l = []
    for x in map(int, input().split()):
        res = 0
        while x:
            res += x & 1
            x >>= 1
        l.append(res)
    print(" ".join(list(map(str, l))))
